ISO C90 compatibility issue

```
gcc -c -Wall -ansi -pedantic -I/usr/include -I../znzlib -I../niftilib fslio.c
fslio.c: In function ‘FslWriteVolumes’:
fslio.c:902:23: error: C++ style comments are not allowed in ISO C90
   size_t retval = 0;  // initialize to failure
```

--- a/fsliolib/fslio.c
+++ b/fsliolib/fslio.c
@@ -899,7 +899,7 @@ void FslWriteAllVolumes(FSLIO *fslio, const void *buffer)
 size_t FslWriteVolumes(FSLIO *fslio, const void *buffer, size_t nvols)
 {
   /* The dimensions and datatype must be set before calling this function */
-  size_t retval = 0;  // initialize to failure
+  size_t retval = 0;  /* initialize to failure */
   if (fslio==NULL)  FSLIOERR("FslWriteVolumes: Null pointer passed for FSLIO");
   if ( (!fslio->written_hdr) && (FslIsSingleFileType(FslGetFileType(fslio))) &&
        (FslIsCompressedFileType(FslGetFileType(fslio))) )

